include/accelerator.h

```diff
@@ -46,7 +46,12 @@ namespace quda
      * @param out Solution vector.
      * @param in Right-hand side.
      */
-    virtual void operator()(ColorSpinorField &out, ColorSpinorField &in)
+    virtual void operator()(cvector_ref<ColorSpinorField> &out, cvector_ref<const ColorSpinorField> &in) override
```

Comment:
Meta-comment, should we start filing issues for things that haven't been natively mrhs'd yet to keep track of outstanding work?

Reply:
Yes, that seems like a good idea.

Reply:
https://github.com/lattice/quda/wiki/MRHS-to-do-list